client/src/App.js

```diff
@@ -0,0 +1,125 @@
+import './App.css';
+import { useState } from 'react';
+import Score from './components/Score';
+import Question from './components/Question';
+import Flag from './components/Flag';
+
+function App() {
+
+  const [submitted, setSubmitted] = useState(false);
+  const [questionChosen, setQuestionChosen] = useState(null);
+  const [questionIndex, setQuestionIndex] = useState(0);
+  const [displayQuestion, setDisplayQuestion] = useState(false);
+  const [resultMessage, setResultMessage] = useState(null);
+  const [score, setScore] = useState(0);
+  const [disable, setDisable] = useState(false);
+  const [flagIndex, setFlagIndex] = useState(1);
+
+  const message = [
```

Comment:
It is more readable if you use an object here, e.g.

```
const message = {
    consolationMessage: 'Sorry your answer is incorrect',
    correctAnswer: 'blah blah',
    partialPuzzleCompletion: 'blo blo',
    successfulCompletion: 'ble ble',
};
```

Reply:
This would be overkill for you and most projects, but if you are working on a bigger team, with an app in multiple languages you might use a localization library: https://phrase.com/blog/posts/react-i18n-best-libraries/